js_modules/dagster-ui/packages/ui-core/src/app/Util.tsx

```diff
@@ -162,8 +162,14 @@ export function indexedDBAsyncMemoize<T, R, U extends (arg: T, ...rest: any[]) =
   const ret = (async (arg: T, ...rest: any[]) => {
     const hashKey = hashFn ? hashFn(arg, ...rest) : arg;
 
+    const encoder = new TextEncoder();
+    const data = encoder.encode(hashKey.toString());
+    const hashBuffer = await crypto.subtle.digest('SHA-1', data);
+    const hashArray = Array.from(new Uint8Array(hashBuffer)); // convert buffer to byte array
+    const hashHex = hashArray.map((b) => b.toString(16).padStart(2, '0')).join(''); // convert bytes to hex string
```

Comment:
Weird we can't use TextDecoder to do this, but it seems to not support hex as an output format!